sync/src/block/extension.rs

```diff
@@ -174,6 +174,10 @@ impl Extension {
     }
 
     fn send_status(&mut self, id: &NodeId) {
+        if std::mem::discriminant(&self.state) != std::mem::discriminant(&State::Full) {
```

Comment:
You can use `if let State::Full = self.state { ... }`

Reply:
Or you can `use std::mem::discriminant`.

Reply:
Okay. I changed that.